pulser-core/pulser/noise_model.py

```diff
@@ -298,7 +296,11 @@ def to_tuple(obj: tuple) -> tuple:
         relevant_param_vals = {
             p: param_vals[p]
             for p in param_vals
-            if param_vals[p] is not None or (p in relevant_params)
+            if not (
+                param_vals[p] is None
+                or (isinstance(param_vals[p], bool) and not param_vals[p])
+            )
+            or p in relevant_params
```

Comment:
I'm not sure, but maybe we can get away with
```suggestion
            if param_vals[p] or p in relevant_params
```
Did you try this?

Reply:
I tried this, you have a misbehaviour with runs=0 (the value of run is not validated, and the NoiseModel is generated with runs=0)

Reply:
Ok, in that case I'd say this special case is not worth it, I'd suggest reverting to the less strict validation that a `bool` is a boolean, not necessarily `True`. In reality, we also allow other parameters to be zero (when not strictly positive), so it's more consistent behaviour

Reply:
I agree, I had the feeling of duplicating the test.